modules/vidazooBidAdapter.js

```diff
@@ -1,6 +1,7 @@
-import * as utils from '../src/utils.js';
-import {registerBidder} from '../src/adapters/bidderFactory.js';
-import {BANNER} from '../src/mediaTypes.js';
+import * as utils from '../src/utils';
+import { registerBidder } from '../src/adapters/bidderFactory';
+import { BANNER } from '../src/mediaTypes';
```

Comment:
Please bring back `.js` extension to these imports.  
Circleci is failing because your imports are missing those extensions.  
Take a look at details from your circleci build.